frontend/src/components/grantaccess/GrantApplicationAdmin.vue

```diff
@@ -0,0 +1,178 @@
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+import { ErrorMessage, Field, Form as VeeForm } from 'vee-validate';
+import { object, string } from 'yup';
+import router from '@/router';
+import { AdminMgmtApiService } from '@/services/ApiServiceFactory';
+import { UserType, type FamApplication } from 'fam-app-acsctl-api';
+import type { FamAppAdminCreateRequest } from 'fam-admin-mgmt-api/model/fam-app-admin-create-request';
+import type { FamApplicationGetResponse } from 'fam-admin-mgmt-api/model/fam-application-get-response';
+import ApplicationSelect from '@/components/common/ApplicationSelect.vue';
+import Button from '@/components/common/Button.vue';
+import { IconSize } from '@/enum/IconEnum';
+import { Severity } from '@/enum/SeverityEnum';
+
+import { isLoading } from '@/store/LoadingState';
+import { setNotificationMsg } from '@/store/NotificationState';
+
+const defaultFormData = {
+    userId: '',
+    application: Number,
```

Comment:
You might be already fixing this...
I saw this "Invalid Prop" when I click cancel button.
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/f9c4533c-3e89-4794-b6d8-cb7ee8f65804)

Reply:
I got the same

Reply:
As the component does not exist anymore, this warning is gone.